src/nvim/log.h

```diff
@@ -67,6 +67,30 @@
        __VA_ARGS__)
 #  endif
 
+# if __linux__
+#  include <execinfo.h>
+#  define LOG_CALLSTACK(prefix) \
```

Comment:
Also if `__linux__` is not defined, `LOG_CALLSTACK` should be defined to something, e.g., `(void)0`.